firefox/manifest.json

```diff
@@ -47,7 +47,8 @@
   },
   "browser_specific_settings": {
     "gecko": {
-      "id": "search@kagi.com"
+      "id": "search@kagi.com",
+      "strict_min_version": "102.0"
```

Comment:
This was the main thing preventing the extension from being compatible with Firefox. This version (102) is 1 year old, so I don't expect people to have problems not being able to install the extension.